storage/store.go

```diff
@@ -1016,9 +1018,12 @@ func (s *Store) processRaft() {
 				nodeID, storeID := decodeRaftNodeID(e.NodeID)
 				cmd.RaftID = groupID
 				ok := cmd.Cmd.SetValue(&proto.InternalChangeReplicasRequest{
-					NodeID:     nodeID,
-					StoreID:    storeID,
-					ChangeType: changeTypeRaftToInternal[e.ChangeType],
+					// TODO(bdarnell): do we need to smuggle the original timestamp through raft?
```

Comment:
I believe this will be correct. Original timestamp doesn't do you any good. The most up to date value here is much better.